js/status.js

```diff
@@ -0,0 +1,234 @@
+(function ($, window)
+{
+  // register namespace
+  $.extend(true, window, {
+    "StatusApp": StatusApp
+    }
+  );
+
+  /**
+   * This application builds and displays a status table constructed from
+   * queries to a list of services. Each row in the table contains
+   * the status of a service.
+   *
+   * param: _options - values to be supplied to this application
+   */
+  function StatusApp(_options)
+  {
+    const VERSION_PARAM = "?version=" + new Date().getTime();
```

Comment:
Please use `var` since older browsers won't handle `const`.  I appreciate wanting to use the new API though.